### PR TITLE
adding line about product team

### PR DESCRIPTION
**Pull request by liliameyers**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# Product_team_test
\ No newline at end of file
+# Product_team_test
```
Comment (mamitch) [inline README.md:1]: Actually could you make this all lowercase?  @liliameyers

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
...
+
+The product team is the best!
```
Comment (mamitch) [inline README.md]: Looks good we are the best!

**Review by mamitch — COMMENTED** [review]
Overall looks good but change a few things.

File: README.md (hunk 1 above)
Comment (liliameyers) [reply README.md:1]: I really agree Marisa!!

File: README.md (hunk 2 above)
Comment (liliameyers) [reply README.md]: Yay

**Review by mamitch — APPROVED** [review]

**Pull request merged**